src/nav/components.yml

```diff
@@ -48,6 +48,8 @@
           title: Toggle icon button
     - title: Chrome
       id: /components/chrome
+    - title: Sheet
+      id: /components/sheet
```

Comment:
Depending on where the team thinks the Sheet component should be placed in the nav, we can move around accordingly.

Reply:
Let's respect current alphabetical ordering.

Reply:
btw, we can see here that "Color picker" and "Color swatch" are out of order – might be nice to sweep those up, too.